### PR TITLE
net: shell: stacks: Print config option needed for full info

### PR DESCRIPTION
**Pull request by pfalcon**
This is similar to how few commands already behave if they can
provide additional info to the user if particular config options
are enabled.

Signed-off-by: Paul Sokolovsky <paul.sokolovsky@linaro.org>

### CODE REVIEW

**Comment by pfalcon** [comment]
Let's continue good tradition of including AI straight into the shell ;-).

**Review by jukkar — CHANGES_REQUESTED** [review]
I do not think this is very useful for the end user.
What about adding general help output after all all the stack prints so something like:
"Enable CONFIG_INIT_STACKS to see usage information."

**Comment by pfalcon** [comment]
I thought about that, but figured it would eat more bytes. Will do as you suggest.

**Comment by pfalcon** [comment]
Updated as suggested.

File: subsys/net/ip/net_shell.c [hunk 1]
```diff
@@ -2168,6 +2168,8 @@ int net_shell_cmd_stacks(int argc, char *argv[])
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE, unused,
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE - unused,
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE, pcnt);
+#else
+	printk("Enable CONFIG_INIT_STACKS to see usage information.");
```
Comment (jukkar) [inline subsys/net/ip/net_shell.c]: This is missing \n at the end of the print.

**Review by jukkar — CHANGES_REQUESTED** [review]

File: subsys/net/ip/net_shell.c (hunk 1 above)
Comment (pfalcon) [reply subsys/net/ip/net_shell.c]: Fixed.

**Review by jukkar — APPROVED** [review]

**Pull request merged**